tests/e2e/conftest.py

```diff
@@ -689,15 +681,25 @@ def nested_data(static_path: Path) -> Tuple[str, str, str]:
 
 
 @pytest.fixture(scope="session")
-def tmp_bucket(tmp_path_factory: Any, request: Any) -> Iterator[str]:
+def tmp_bucket_create(
```

Comment:
Hint: `pytest` assumes that fixtures started with an underscore are *private*, they are not displayed by `pytest --fixtures` etc.
Maybe this is a good convention for `tmp_bucket_create`?